examples/v1/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 	}
 	defer fh.Close()
 
-  fi, err := fh.Stat()
+	fi, err := fh.Stat()
```

Comment:
...and likewise, it'd be nice for this to be folded back into b3d20797fd7fe77aa677f308022984313c2fe430.

Reply:
squashed in 808952d2d3